requirements.txt

```diff
@@ -5,3 +5,4 @@ redis
 flask-wtf
 flask-sqlalchemy
 stravalib==0.9
+pyquery
```

Comment:
Tip: remember to run `pip install -r requirments.txt` to download the `pyquery` lib.